app/main.py

```diff
@@ -1,6 +1,23 @@
+import os
+import requests
+
+
 def get_weather() -> None:
-    # write your code here
-    pass
+    api_key = os.getenv("API_KEY")
+    if not api_key:
+        return "Failed to get api key"
```

Comment:
Consider raising an exception or logging an error message instead of returning a string when the API key is missing. This will make it easier to debug issues related to missing environment variables.